### ISSUE TITLE
Precompilation issue on Apple M1

### ISSUE THREAD

**Issue by blegat** [kalmarek/SymbolicWedderburn.jl#87]
See https://github.com/jump-dev/SumOfSquares.jl/issues/382
cc @kocvara

**Comment by kalmarek** [comment]
yeah, I don't even know what is the problem. There's no fancy BLAS happening, except `using LinearAlgebra` and a call to `qr` or `svd`. If the problem is that there's no default BLAS implementation set I don't think we should set one for the user?

**Comment by blegat** [comment]
Maybe this is because of code called before `using SymbolicWedderburn`.
@kocvara Can you reproduce it by running `using SymbolicWedderburn` on a new Julia REPL to make sure it's not due to code run before like what's suggested in
https://github.com/jump-dev/SumOfSquares.jl/issues/382#issuecomment-2297759194

**Comment by odow** [comment]
This issue can be closed. I can't reproduce it, and it seems a general problem with @kocvara's setup, rather than something to do with this package.
